pkg/updater/service/helper/main.go

```diff
@@ -23,14 +23,29 @@ import (
 )
 
 var (
-	installPath string
-	systemdPath = "/lib/systemd/system" // todo load it at build time from omnibus
-	pkgDir      = "/opt/datadog-packages"
-	agentDir    = "/etc/datadog-agent"
-	dockerDir   = "/etc/docker"
-	testSkipUID = ""
+	installPath    string
+	debSystemdPath = "/lib/systemd/system" // todo load it at build time from omnibus
+	rpmSystemdPath = "/usr/lib/systemd/system"
+	pkgDir         = "/opt/datadog-packages/"
+	agentDir       = "/etc/datadog-agent"
+	dockerDir      = "/etc/docker"
+	testSkipUID    = ""
+	installerUser  = "dd-installer"
 )
 
+// findSystemdPath todo: this is a hacky way to detect on which os family we are currently
+// running and finding the correct systemd path.
+// We should probably provide the correct path when we build the package
+func findSystemdPath() (systemdPath string, err error) {
+	if _, err = os.Stat(rpmSystemdPath); err == nil {
```

Comment:
nothing prevents this path to be added by another library over time, making the rm fail after, same for the load systemd path
You might want to run a  (load|rm) path1 || path2, or enforce it